tsconfig.json

```diff
@@ -18,11 +19,13 @@
     "baseUrl": ".",
     "paths": {
       "app.constants": ["core/templates/app.constants"],
+      "app-events/*": ["core/templates/app-events/*"],
```

Comment:
This is added to array path in `tsconfig-strict.json` but not here. So, added here to maintain consistency.